aws/rakefiles/deploy.rake

```diff
@@ -80,7 +80,7 @@ task :find_gpii_components => :generate_modules do
 end
 
 desc "Deploy GPII components to cluster"
-task :deploy => [:apply, :configure_kubectl, :wait_for_cluster_up, :init_helm, :find_gpii_components] do
+task :deploy => [:apply, :configure_kubectl, :wait_for_cluster_up, :setup_rbac, :init_helm, :find_gpii_components] do
```

Comment:
I guess :init_helm should also depend on :setup_rbac.

Reply:
Makes sense